Assets/_Scripts/Character/StatusAilment/ModifierAilment.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+// An ailment that is used only for its name
+[CreateAssetMenu(fileName = "ModifierAilment", menuName = "StatusAilment/ModifierAilment")]
+public class ModifierAilment : StatusAilment {
+
+    // TODO: move this to somewhere more appropriate
+    public const string MODIFIER_DEATH = "death";
+    public const string MODIFIER_UNTARGETTABLE = "untargetable";
+    public const string MODIFIER_CANNOT_USE_ACTION = "cannot move";
```

Comment:
I like this idea about player state, but not sure if it should be built into Ailments. We can keep this for now.

Reply:
Originally I just made it as a hashset in Player, but I thought it made more sense to make it an ailment, because it essentially does the same thing.